src/interface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+interface IShapeDrawRectangle
+{
+  public function checkSidesValue($height, $width);
```

Comment:
I think checkSidesValue, getValueSide, setValueSide could be moved to interface IShape

Reply:
I can't do that now.
checkIfSidesMoreThanZero
getValueSide
setValueSide
these methods now have different names and different arguments